UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformationAdmin.Web/Controllers/RadioNavigationalWarningsAdminController.cs

```diff
@@ -39,16 +39,30 @@ public async Task<IActionResult> Create(RadioNavigationalWarning radioNavigation
         {
             _logger.LogInformation(EventIds.CreateNewRNWRecordStart.ToEventId(), "Create RNW request started for _X-Correlation-ID:{correlationId}. Requested by user: {user}", GetCurrentCorrelationId(),User.Identity.Name);
 
+            bool skipCheckDuplicateReference = false;
+            if (!string.IsNullOrWhiteSpace(Request.Form["SkipCheckDuplicate"]))
+            {
+                skipCheckDuplicateReference = Request.Form["SkipCheckDuplicate"] == "Yes";
+            }
+
             if (ModelState.IsValid)
             {
-                bool result = await _rnwService.CreateNewRadioNavigationWarningsRecord(radioNavigationalWarning, GetCurrentCorrelationId());
+                bool result = await _rnwService.CreateNewRadioNavigationWarningsRecord(radioNavigationalWarning, GetCurrentCorrelationId(), skipCheckDuplicateReference, User.Identity.Name);
```

Comment:
Better rename as isNewRecordCreated for readability.
```suggestion
                bool isNewRecordCreated = await _rnwService.CreateNewRadioNavigationWarningsRecord(radioNavigationalWarning, GetCurrentCorrelationId(), skipCheckDuplicateReference, User.Identity.Name);
```

Reply:
Done changes.